canonical/operator
# Fix issue with relative paths in Container.push_path

This PR fixes the error caused by duplicate paths in `container.push_path()`.

Please see #948 .

## QA steps

*Please replace with how we can verify that the change works.*

```sh
QA steps here
Running `tox`
```

## Documentation changes

No changes required. 

## Bug reference

Fixes #948

## Changelog

- Fixes bug in container.push_path, which caused FileNotFoundError when pushing paths to the container.
